mapfishapp/src/main/webapp/app/js/GEOR.js

```diff
@@ -142,12 +143,17 @@ Ext.namespace("GEOR");
         Proj4js.libPath = GEOR.config.PATHNAME + "/lib/proj4js/lib/";
         Ext.apply(Proj4js.defs, GEOR.config.PROJ4JS_STRINGS);
 
+        // State manager for the viewer
+        Ext.state.Manager.setProvider(new Ext.state.LocalStorageProvider({
+            prefix: "geor-viewer-"
+        }));
+
         /*
          * Security stuff
          * Deactivate modules if current user roles do not match
          */
         checkRoles('styler', GEOR.config.ROLES_FOR_STYLER);
-        checkRoles('querier', GEOR.config.ROLES_FOR_QUERIER);
+        checkRoles('Querier', GEOR.config.ROLES_FOR_QUERIER);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["GEOR" is not defined.](https://www.codacy.com/app/georchestra/georchestra/file/2338931039/issues/source?bid=3200181&fileBranchId=3209168#l156)